### PR TITLE
#11333 postgres connection still using old database name when config …

### PR DESCRIPTION
**Pull request by dextok**
Fixed postgres using database name form old instance when open new connection

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreDataSource.java [hunk 1]
```diff
@@ -376,7 +376,7 @@ protected Connection openConnection(@NotNull DBRProgressMonitor monitor, @Nullab
             try {
                 // Patch URL with new database name
                 if (CommonUtils.isEmpty(conConfig.getUrl()) || !CommonUtils.isEmpty(conConfig.getHostName())) {
-                    conConfig.setDatabaseName(instance.getName());
+                    //conConfig.setDatabaseName(instance.getName());
```
Comment (samuei) [inline plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreDataSource.java]: Let's not leave commented code laying around. If we're removing a line, let's remove the line.

Comment (dextok) [reply plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreDataSource.java]: It has been removed, thanks for your review.

Comment (samuei) [reply plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreDataSource.java]: 👍

**Comment by serge-rider** [comment]
Will get back to it after version 21 release

**Pull request closed without merging**